payjoin/src/send/v2/mod.rs

```diff
@@ -354,6 +377,14 @@ impl V2GetContext {
         Ok((Request::new_v2(&url, &body), ohttp_ctx))
     }
 
+    /// Processes the response for the final GET message from the sender client in the v2 Payjoin protocol.
+    ///
+    /// This function decapsulates the response using the provided OHTTP context.
+    /// A successful response can either be a Proposal PSBT or an ACCEPTED message indicating no Proposal PSBT is available yet.
+    /// Otherwise, it returns an error with the status code.
+    ///
+    /// After this function is called, the sender can sign and finalize the PSBT and broadcast the
+    /// resulting Payjoin transaction to the network.
```

Comment:
similar encapsulation concern re: status code. ACCEPTED message / Proposal PSBT seem less ambiguous.

```suggestion
    /// Processes the response for the final GET message from the sender client
    /// in the v2 Payjoin protocol.
    ///
    /// This function decapsulates the response using the provided OHTTP
    /// context. A successful response can either be a Proposal PSBT or an
    /// ACCEPTED message indicating no Proposal PSBT is available yet.
    /// Otherwise, it returns an error with the encapsulated status code.
    ///
    /// After this function is called, the sender can sign and finalize the
    /// PSBT and broadcast the resulting Payjoin transaction to the network.
```